src/commands/org/create/scratch.ts

```diff
@@ -86,6 +84,7 @@ export default class EnvCreateScratch extends SfCommand<ScratchCreateResponse> {
     'duration-days': Flags.duration({
       unit: 'days',
       defaultValue: 7,
+      required: true,
```

Comment:
Why is this required?

Reply:
good point.  Setting the default the "fixed" way seems to do it better (make it clear that it can't be undefined).  

that didn't used to work correctly, but got fixed (while Mike was in last, I think?)